pyvista/plotting/render_window_interactor.py

```diff
@@ -170,9 +170,23 @@ def remove_observers(self, event=None):
         for observer in observers:
             self.remove_observer(observer)
 
-    def clear_events_for_key(self, key):
-        """Remove the callbacks associated to the key."""
-        self._key_press_event_callbacks.pop(key)
+    def clear_events_for_key(self, key, raise_on_missing=False):
+        """Remove the callbacks associated to the key.
+
+        Parameters
+        ----------
+        key : str
+            Key to clear events for.
+
+        raise_on_missing : bool, default=False
```

Comment:
I forgot that the final decision was the other one
```suggestion
        raise_on_missing : bool, default: False
```